string-manipulation-2/src/capitalize-words.js

```diff
@@ -0,0 +1,11 @@
+/* exported capitalizeWords */
+function capitalizeWords(string) {
+  var k = [];
```

Comment:
Please stop using single letters for your variable names, with the exception of `i`. Your variables should be named to describe their purpose, _always_